frontend-web/webclient/app/Applications/Studio/Applications.tsx

```diff
@@ -244,17 +201,23 @@ export const App: React.FunctionComponent = () => {
                                 refresh();
                             }}
                         >
-                            <Heading.h2>Group</Heading.h2>
-                            <Flex>
-                                <GroupSelector
-                                    selectedGroup={selectedGroup}
-                                    applicationName={name}
-                                    options={allGroups}
-                                    onSelect={item => setSelectedGroup(item)}
-                                />
-                            </Flex>
+                            <Label>Group</Label>
+                            <Box mb="20px" textAlign="right">
+                                    <Select
+                                        name="group" 
+                                        value={selectedGroup}
+                                        onChange={(event) => setSelectedGroup(event.target.value)}
+                                    >
+                                        {groupOptions.map(it => (
+                                            <option value={it.value}>{it.text}</option>
+                                        ))}
```

Comment:
Small thing, this selects the first option by default, right? Would it make sense to have an empty options, so UCloud Admins don't select the first group by mistake?

Reply:
Yes, I actually thought about it, but forgot to add it :+1: